python/tests/sklearn/base_tests.py

```diff
@@ -31,9 +31,28 @@
 from mleap.sklearn.preprocessing.data import FeatureExtractor, Binarizer
 
 
+def to_quartile(rand_num):
```

Comment:
One thing, this is very specific to a standard normal distribution (mean 0 and variance 1?). Could this be reflected in the naming?